apps/server/src/modules/tool/context-external-tool/service/restricted-context-mismatch-loggabble.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { ToolContextType } from '../../common/enum';
+import { RestrictedContextMismatchLoggable } from './restricted-context-mismatch-loggabble';
+
+describe('RestrictedContextMismatchLoggable', () => {
+	describe('constructor', () => {
```

Comment:
constructor tests like this don't make sense. With this you are testing that the typescript compiler works, by checking if it can create objects. You are not actually testing your own code.

Reply:
done! 180 lines deleted :D